packages/vapor/scss/tables/table.scss

```diff
@@ -176,6 +176,37 @@
     &.mod-deleting tbody tr.selected td {
         background-color: rgba($red, 0.1);
     }
+
+    .mod-card-list {
```

Comment:
Are you missing the `&` to scope the class on the table?
```suggestion
    &.mod-card-list {
```

Reply:
From our example in BlacklistsConfiguration, we might need `.table` to be the parent of `.mod-card-list`, since `mod-card-list` would be applied to `<tbody>`.
![image](https://user-images.githubusercontent.com/52677246/74958322-c287ae00-53d6-11ea-9ff1-0eed11af647b.png)